pyaerocom/aeroval/modelmaps_helpers.py

```diff
@@ -146,3 +103,41 @@ def calc_contour_json(data, cmap, cmap_bins):
 
     plt.close("all")
     return geojson
+
+
+def plot_overlay_pixel_maps(
+    data: xarray.DataArray, cmap: str, cmap_bins: list[float], format: str
+):  # pragma: no cover
+    plt.close("all")
+    matplotlib.use("Agg")
+
+    fig, axis = plt.subplots(
+        1,
+        1,
+        subplot_kw=dict(projection=ccrs.Mercator()),
```

Comment:
Why are there two projections, Mercator and PlateCarree in this function? We should only have one output projection.

Reply:
The `transform` argument tells cartopy which projection our data is in. The `projection` argument tells cartopy which projection to project into. https://scitools.org.uk/cartopy/docs/latest/tutorials/understanding_transform.html